tasks/lint-all.js

```diff
@@ -25,9 +25,13 @@ fluid.grunt.lintAll.defaults = {
         }
     },
     eslint: {
+        options: {
+            format: fluid.module.resolvePath("%fluid-grunt-lint-all/formatters/stylish/stylish")
+        },
         md: {
             options: {
-                configFile: fluid.module.resolvePath("%fluid-grunt-lint-all/.eslintrc-md.json")
+                configFile: fluid.module.resolvePath("%fluid-grunt-lint-all/.eslintrc-md.json"),
+                useEslintrc: false
```

Comment:
Once you file a ticket about enabling eslintrc support, please add a TODO here referencing it.